mongo/read_write_concern_spec_test.go

```diff
@@ -103,9 +104,7 @@ func runConnectionStringTestFile(t *testing.T, filePath string) {
 }
 
 func runConnectionStringTest(t *testing.T, test connectionStringTest) {
-	if test.SkipReason != "" {
```

Comment:
Can we also remove the `SkipReason` field from the `connectionStringTest` struct? As far as I can tell, the [connection-string test spec](https://github.com/mongodb/specifications/blob/6118debee41cfd1bca197b315bd1f10ad95f66ae/source/connection-string/tests/README.md) don't include that field.